collector/README.md

```diff
@@ -88,4 +88,70 @@ from an S3 object using a CloudFormation template:
           OPENTELEMETRY_COLLECTOR_CONFIG_FILE: s3://<bucket_name>.s3.<region>.amazonaws.com/collector_config.yaml
 ```
 
-Loading configuration from S3 will require that the IAM role attached to your function includes read access to the relevant bucket.
\ No newline at end of file
+Loading configuration from S3 will require that the IAM role attached to your function includes read access to the relevant bucket.
+
+# Improving Lambda responses times
+At the end of a lambda function's execution, the OpenTelemetry client libraries will flush any pending spans/metrics/logs
+to the collector before returning control to the Lambda environment. The collector's pipelines are synchronous and this 
+means that the response of the lambda function is delayed until the data has been exported. 
+This delay can potentially be for hundreds of milliseconds.
+
+To overcome this problem the [decouple](./processor/decoupleprocessor/README.md) processor can be used to separate the 
+2 ends of the collectors pipeline, allow the lambda function to complete and still ensuring that any data is exported 
```

Comment:
```suggestion
two ends of the collectors pipeline and allow the lambda function to complete while ensuring that any data is exported 
```
I started the edit here just to adjust the phrase "and still ensuring" to fit with the context. Ended up spelling **two** just because that's how the style guides I know of say to do it.